packages/core/content-type-builder/package.json

```diff
@@ -21,21 +21,29 @@
   ],
   "exports": {
     "./strapi-admin": {
-      "source": "./admin/src/index.js",
+      "types": "./dist/admin/index.d.ts",
+      "source": "./admin/src/index.ts",
       "import": "./dist/admin/index.mjs",
       "require": "./dist/admin/index.js",
       "default": "./dist/admin/index.js"
     },
     "./strapi-server": {
-      "source": "./strapi-server.js",
-      "require": "./strapi-server.js",
-      "default": "./strapi-server.js"
+      "types": "./dist/server/src/index.d.ts",
+      "source": "./server/src/index.ts",
+      "import": "./dist/server/index.mjs",
+      "require": "./dist/server/index.js",
+      "default": "./dist/server/index.js"
     },
     "./package.json": "./package.json"
   },
+  "types": "./dist/index.d.ts",
+  "files": [
+    "./dist",
+    "strapi-server.js"
+  ],
   "scripts": {
```

Comment:
Can you add `"test:ts:front": "run -T tsc -p admin/tsconfig.json",` otherwise we're not type-checking the code in the CI. I would also add the be equivalent imo